nucliadb/tests/nucliadb/integration/test_ask.py

```diff
@@ -224,9 +224,12 @@ async def test_sync_ask_returns_debug_mode(
         assert resp_data.prompt_context
         assert resp_data.predict_request
         assert isinstance(resp_data.predict_request, dict)
+        assert isinstance(resp_data.debug, dict)
+        assert resp_data.debug["metrics"]
```

Comment:
Maybe check that it actually contains some values as well? e.g: Check that `index_search` is > 0 or something like that. This way we make sure we are actually passing some data and not just an empty dict.